content/rs/references/client_references/client_ioredis.md

```diff
@@ -0,0 +1,110 @@
+---
+Title: Using Redis with Node.js (ioredis)
+description:
+weight:
+alwaysopen: false
+categories: ["RS"]
+---
+In order to use Redis with Node.js, you need to install a Node.js Redis client.  In the following sections, we demonstrate the use of [ioredis](https://github.com/luin/ioredis), a community recommended Redis client for Node.js with support for Promises built in.  The other community recommended client for Node.js developers is [node_redis]({{< relref "./client_nodejs" >}}).
+
+Additional Node.js clients for Redis can be found under the [Node.js section](https://redis.io/clients#Node.js) of the [Redis Clients page](https://redis.io/clients).
+
+## Installing ioredis
+
+Complete instructions for installing ioredis can be found in the [README file](https://github.com/luin/ioredis/blob/master/README.md). To install ioredis, issue the following command:
+
+    npm install ioredis 
+
+## Connecting to Redis
+
+The following code creates a connection to Redis:
+
+    const Redis = require('ioredis');
+    const redis = new Redis({
+      host: 'hostname',
+      port: 6379,
+      password: 'password'
+    });
+
+Make sure to replace the following values with those for your Redis instance:
+
+- `'hostname'` should be the name of the host your database runs on
+- `6379` is the default Redis port, change this if necessary
+- `'password'` should be the password you use to access Redis.  If a password is not required, remove this line.  Remember to always store passwords outside of your code, for example in environment variables
+
+## Connecting to Redis with TLS 
+
+The following example demonstrates how to configure ioredis to make a connection to Redis using TLS:
+
+    const Redis = require('ioredis');
+    const fs = require('fs');
+
+    const redis = new Redis({
+        host: 'hostname',
+        port: 6379,
+        tls: {
+            key: fs.readFileSync('path_to_keyfile', encoding='ascii'),
+            cert: fs.readFileSync('path_to_certfile', encoding='ascii'),
+            ca: [ fs.readFileSync('path_to_ca_certfile', encoding='ascii') ]
+        }
+    });
+    
+    ...
```

Comment:
Are the ... just to show the continuation of the code?

Reply:
Yes